mesonbuild/compilers/compilers.py

```diff
@@ -459,6 +459,15 @@ def get_base_link_args(options, linker, is_shared_module):
     elif as_needed:
         # -Wl,-dead_strip_dylibs is incompatible with bitcode
         args.append(linker.get_asneeded_args())
+    try:
+        crt_val = options['b_vscrt'].value
+        buildtype = options['buildtype'].value
+        try:
+            args += linker.get_crt_link_args(crt_val, buildtype)
+        except AttributeError:
```

Comment:
This should be implemented in the same way as other arg functions. There should be a method of the same name in the base class that returns `[]`. This way there does not need to be any special error handling here.

Reply:
I just mirrored how it was done with `get_crt_compile_args`, so surely that should be fixed as well?